test/subscriptions.js

```diff
@@ -7,6 +7,7 @@ const assert = require("assert");
 const PORT = 8545;
 const HOST = "127.0.0.1";
 const HTTPADDRESS = `http://${HOST}:${PORT}`;
+var Web3WsProvider = require("web3-providers-ws");
```

Comment:
Since `web3-providers-ws` isn't in our package.json it shouldn't be used here (I know we already have tests that do this - shame on them!). Either included it in package.json or use `Web3.providers.WebsocketProvider` instead (which is what I prefer)